org-code-javabuilder/protocol/src/main/java/org/code/protocol/LoggerUtils.java

```diff
@@ -101,17 +103,29 @@ public static void logSevereException(Throwable e) {
    */
   public static void logTrackingException(Throwable e) {
```

Comment:
Could we go ahead and change this method name to something warning specific? maybe something like `logExceptionAsWarning` or even just another `logWarning`?

Reply:
good idea, will update